resources/templates/ha-vpn-interop-guide-template/index.md

```diff
@@ -0,0 +1,836 @@
+---
+title: Google Cloud HA VPN Interop Guide for [PRODUCT]
+description: Describes how to build site-to-site IPsec VPNs between HA VPN on Google Cloud Platform (GCP) and [VENDOR] [PRODUCT]
+author: [AUTHOR]
+tags: HA VPN, Cloud VPN, interop, [VENDOR], [PRODUCT]
+date_published: YYYY-mm-dd
+---
+
+**BEGIN: HOW TO USE THIS TEMPLATE**
+
+1. Make a copy of this template.
+1. On your local computer, update and add information as indicated. Note:
+    + Fill in the meta information (title, description, author, date_published at the top
+      of this file.
+    + There are notes in the template for you, the author, that explain where you need to
+      make changes. These notes are enclosed in brackets (&lt; &gt;).
+    + The template contains placeholders for things like the vendor and 
+      product name. These are also enclosed in brackets—for example, 
+      every place you see `<vendor-name>` and `<product-name>`, 
+      substitute approriate names.
+    + After you've made appropriate updates, _remove_ bracketed content.
+    + Remove these instructions.
+    + Because this is a template for a variety of setups, it might contain
+      content that isn't releavent to your scenario. Remove (or update)
+      any sections that don't apply to you.
+1. Fork the [GoogleCloudPlatform/community/](https://github.com/GoogleCloudPlatform/community/) repo.
+4. In your fork, add a new folder named `/tutorials/[YOUR_TUTORIAL]`. For the
+   folder name, use hyphens to separate words. We recommend that you 
+   include a product name in the folder name, such as `https-load-balancing-ingix`.
```

Comment:
Possibly you should add the directory name of a VPN tutorial instead of `https-load-balancing-ingix.`

Reply:
Done